modules/core/src/main/java/org/apache/ignite/internal/management/snapshot/SnapshotStatusCommand.java

```diff
@@ -68,23 +70,36 @@ public class SnapshotStatusCommand extends AbstractSnapshotCommand<NoArg> {
             throw new IgniteException(e);
         }
 
-        if (res == null) {
-            printer.accept("There is no create or restore snapshot operation in progress.");
```

Comment:
I'm afraid we must not change this output. Logs are part of public API, and this can be used by users for any scripts. For example, user could have a script for validate that there is no operations in progress before starting a snapshot creation. This script might use `grep` for this line. Now this script starts failing.

Reply:
Fixed